tests/test_admin2.py

```diff
@@ -81,5 +81,5 @@ def test_code_date_unique(run_constraints_test):
                 hapi_replaced_date=None,
             ),
         ],
-        expected_constraint="UNIQUE constraint failed",
+        expected_constraint="admin2_code_hapi_updated_date",
```

Comment:
I couldn't get the unique constraint names to show up in the logs, how did you do it?

Reply:
With a named unique constraint, it worked in postgres so it's probably just a difference with SQLite.